<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Diploma Generator</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>981</width>
     <height>581</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_main">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_text_menu">
      <item>
       <widget class="QGroupBox" name="groupBox_text">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>250</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>250</height>
         </size>
        </property>
        <property name="title">
         <string>Text</string>
        </property>
        <widget class="QWidget" name="verticalLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>461</width>
           <height>215</height>
          </rect>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_text">
          <item>
           <widget class="QLineEdit" name="lineEdit_text">
            <property name="text">
             <string>Some Simple Text</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_font">
            <item>
             <layout class="QFormLayout" name="formLayout_font">
              <item row="0" column="0">
               <widget class="QLabel" name="label_font">
                <property name="text">
                 <string>Font</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_size">
                <property name="text">
                 <string>Size</string>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_style">
                <property name="text">
                 <string>Style</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QLineEdit" name="lineEdit_font">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QLineEdit" name="lineEdit_size">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QLineEdit" name="lineEdit_style">
                <property name="enabled">
                 <bool>false</bool>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="pushButton_edit_font">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>70</height>
               </size>
              </property>
              <property name="text">
               <string>Edit Font</string>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_pos_col">
            <item>
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="topMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_position">
                <property name="minimumSize">
                 <size>
                  <width>0</width>
                  <height>0</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>33</height>
                 </size>
                </property>
                <property name="text">
                 <string>Position</string>
                </property>
                <property name="scaledContents">
                 <bool>false</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                </property>
               </widget>
              </item>
              <item>
               <layout class="QGridLayout" name="gridLayout" columnstretch="0,0,0">
                <property name="sizeConstraint">
                 <enum>QLayout::SetFixedSize</enum>
                </property>
                <property name="horizontalSpacing">
                 <number>6</number>
                </property>
                <property name="verticalSpacing">
                 <number>10</number>
                </property>
                <item row="1" column="1">
                 <widget class="QSpinBox" name="spinBox_y">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="minimum">
                   <number>-100000</number>
                  </property>
                  <property name="maximum">
                   <number>100000</number>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QSpinBox" name="spinBox_x">
                  <property name="enabled">
                   <bool>false</bool>
                  </property>
                  <property name="minimum">
                   <number>-100000</number>
                  </property>
                  <property name="maximum">
                   <number>100000</number>
                  </property>
                 </widget>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_x">
                  <property name="text">
                   <string>x (pixels) : </string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QCheckBox" name="checkBox_x">
                  <property name="text">
                   <string>Center</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                  <property name="tristate">
                   <bool>false</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="2">
                 <widget class="QCheckBox" name="checkBox_y">
                  <property name="text">
                   <string>Center</string>
                  </property>
                  <property name="checked">
                   <bool>true</bool>
                  </property>
                 </widget>
                </item>
                <item row="1" column="0">
                 <widget class="QLabel" name="label_y">
                  <property name="text">
                   <string>y (pixels) : </string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QFormLayout" name="formLayout_color">
              <item row="0" column="0">
               <widget class="QLabel" name="label_colour">
                <property name="maximumSize">
                 <size>
                  <width>16777215</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="text">
                 <string>Colour</string>
                </property>
               </widget>
              </item>
              <item row="0" column="1">
               <widget class="QPushButton" name="pushButton_setcolour">
                <property name="text">
                 <string>Set</string>
                </property>
               </widget>
              </item>
              <item row="1" column="0">
               <widget class="QLabel" name="label_red">
                <property name="text">
                 <string>Red :</string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="2" column="0">
               <widget class="QLabel" name="label_greem">
                <property name="text">
                 <string>Green : </string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item row="3" column="0">
               <widget class="QLabel" name="label_blue">
                <property name="text">
                 <string>Blue : </string>
                </property>
                <property name="alignment">
                 <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
              <item row="1" column="1">
               <widget class="QSpinBox" name="spinBox_red">
                <property name="maximum">
                 <number>255</number>
                </property>
               </widget>
              </item>
              <item row="2" column="1">
               <widget class="QSpinBox" name="spinBox_green">
                <property name="maximum">
                 <number>255</number>
                </property>
               </widget>
              </item>
              <item row="3" column="1">
               <widget class="QSpinBox" name="spinBox_blue">
                <property name="maximum">
                 <number>255</number>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_image">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>45</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>65</height>
         </size>
        </property>
        <property name="title">
         <string>Image</string>
        </property>
        <widget class="QWidget" name="gridLayoutWidget">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>30</y>
           <width>461</width>
           <height>26</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_backimage">
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButton_getback">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset theme="folder">
              <normaloff>.</normaloff>.</iconset>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_backimage">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label_backimage">
            <property name="text">
             <string>Bacground Image</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_storage">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>65</height>
         </size>
        </property>
        <property name="title">
         <string>Storage</string>
        </property>
        <widget class="QWidget" name="gridLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>461</width>
           <height>41</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_folder">
          <item row="1" column="1">
           <widget class="QLabel" name="label_folder">
            <property name="text">
             <string>Folder to saving  </string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLineEdit" name="lineEdit_folder">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="3">
           <widget class="QPushButton" name="pushButton_get_folder">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>65</height>
         </size>
        </property>
        <property name="title">
         <string>List of names</string>
        </property>
        <widget class="QWidget" name="gridLayoutWidget_3">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>461</width>
           <height>31</height>
          </rect>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QLabel" name="label_names">
            <property name="text">
             <string>Names  (*.TXT)  </string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEdit_names">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButton_names">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_actions">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <property name="title">
         <string>Actions</string>
        </property>
        <widget class="QWidget" name="horizontalLayoutWidget_2">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>461</width>
           <height>80</height>
          </rect>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_actions">
          <item>
           <widget class="QPushButton" name="pushButton_preview">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="sizePolicy">
             <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>65</height>
             </size>
            </property>
            <property name="text">
             <string>Preview</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_generate">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>65</height>
             </size>
            </property>
            <property name="text">
             <string>Generate</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="groupBox_preview">
      <property name="title">
       <string>Preview</string>
      </property>
      <widget class="QLabel" name="label_preview">
       <property name="geometry">
        <rect>
         <x>16</x>
         <y>22</y>
         <width>451</width>
         <height>541</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>preview image</string>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
